src/list.js

```diff
@@ -407,8 +407,14 @@ Sk.builtin.list.prototype.list_sort_ = function (self, cmp, key, reverse) {
     var timsort;
     var has_key = key !== undefined && key !== null;
     var has_cmp = cmp !== undefined && cmp !== null;
+    var rev;
+
     if (reverse === undefined) {
-        reverse = false;
+        rev = false;
+    } else if (reverse === Sk.builtin.none.none$) {
+        throw new Sk.builtin.TypeError("an integer is required");
+    } else {
+        rev = Sk.ffi.remapToJs(Sk.builtin.bool(reverse));
```

Comment:
This is correct, but would it be better to just use ```Sk.misceval.isTrue``` directly here?  That saves a round trip mapping to Python and back to Javascript.  Sorry for not pointing that out earlier...